tests/Dockerfile

```diff
@@ -11,11 +11,11 @@ RUN dnf install -y bc git go-toolset python3-pip unzip && \
     git clone https://github.com/crobby/peak $HOME/peak && \    
     cd $HOME/peak && \
     git submodule update --init
-
-RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
+RUN echo "Installing chrome"
```

Comment:
Do you still need this echo here?

Reply:
doh

Reply:
just making sure that someone was paying attention this morning...good work